journals/apps/journals/models.py

```diff
@@ -298,6 +299,20 @@ def data(self):
         self.file.close()
         return contents
 
+    def get_viewer_url(self, base_url):
+        '''
+        Return full url to document viewer for this document
+        '''
+        if self.is_pdf():
+            return urljoin(base_url, 'static/pdf_js/web/viewer.html?file={document_path}'.format(
+                document_path=self.file.url)  # pylint: disable=no-member
+            )
+        else:
+            return urljoin(base_url, '{document_path}'.format(document_path=self.file.url))  # pylint: disable=no-member
```

Comment:
Isn't `'{document_path}'.format(document_path=self.file.url)` the same as `self.file.url`?

Reply:
fixed

Reply:
Did you push it?